### ISSUE TITLE
Create overview visualization of EDGI money flow

### ISSUE THREAD

**Issue by shapironick** [edgi-govdata-archiving/fiscal#1]
![edgi finances overview 9 2](https://user-images.githubusercontent.com/7503343/28328046-35e20bf2-6bb3-11e7-98cb-8b7dcc71c190.jpg)

Here is an initial image of how money moves through EDGI. 

I created it using draw.io and [here](https://www.dropbox.com/s/b4ln7hm7o9ssonr/EDGI%20Finances%20Overview%20%289%29%20%285%29.xml?dl=0) is the current xml

**Comment by shapironick** [comment]
to do: add in LD and UCSC as direct support for C & G. Also maybe segregating between AB, TR and RB's  work as one is squarely web monitoring and the other two are largely web monitoring but also seemingly some C&G work (related to FIOA).

**Comment by dcwalk** [comment]
This is super cool!  (and was really illuminating for me)

Thoughts on visualization:
- Might be nice to have a legend/key for different arrow colours/types
- is arrow thickness tied to amt? That could be really subtle but helpful cue (in line with [sankey diagram ](https://en.wikipedia.org/wiki/Sankey_diagram) flows?
- In addition to flow, some tabular summary might be helpful
- It could be good to distinguish payment description text (e.g., direct TRU payments) from the EDGI payment area? (e.g, Administrative)

And then a couple questions: 
-what does green text v. red for dollar amounts mean?
-Is "Public Lab Total" the amount that has gone through PL? (if so maybe there is a word other than 'total' to represent that?)

**Comment by shapironick** [comment]
Thanks @dcwalk for the great feedback!
I've tried to respond to all of your feedback in this current version. Does this seem ok? 
![edgi finances overview 9 5 1 2](https://user-images.githubusercontent.com/7503343/28596707-25b6b6d2-7168-11e7-8911-a3fdd7cead16.jpg)

[Here ](https://www.dropbox.com/s/1k673rb4s3655m8/EDGI%20Finances%20Overview%20%289%29%20%285%29%20%281%29%20%284%29.xml?dl=0)is the file. you can just upload it to draw.io to make changes as well.

**Comment by Blackglade** [comment]
just a suggestion, but instead of making a dynamic javascript visualisation, it might make more sense to just display all this info on a page and vastly simplify it... like a live updating infograph. 

Something along the lines of [THIS](https://www.cardsagainsthumanity.com/holidaystats/). It would be much easier to maintain and update.

**Comment by dcwalk** [comment]
Good point @Blackglade !

Maybe eventually we could set up a static page like we do for the [100 days report](http://100days.envirodatagov.org/) (e.g., `fiscal.envirodatagov.org`), though if it is static it will be much easier to have on the wordpress site (e.g., `envirodatagov.org/fiscal) I'm going to move this convo to a new issue tho, as I am getting ahead of myself and I think it would be good to iterate on display here?

**Comment by dcwalk** [comment]
![there-was-an-attempt](https://user-images.githubusercontent.com/4692834/28894309-6ec3462a-77a2-11e7-8d0b-0c542b63dfc4.jpg)

I was procrastinating and gave it a go. Just realized that draw.io allows saving to GitHub, why don't we store the drafts in here? I've opened a PR with my version so there is access to that file while we iterate, but please @shapironick add yours in if I've gone too far down a tangent

**Comment by shapironick** [comment]
Oh this is is great @dcwalk ! thanks for the sprucing and clarifying. I think it would be ideal to also have disclosure of scale of payments made by collaborators like TRU, USC, QRI etc for this to be complete. I'm going to also try to add individual donations on there as well. 

Great that you initiated the PR. i wasn't sure of the security issues involved in giving draw.io access to my account and our repo so was holding off on doing the pr. but glad that seems ok! 

Also thanks @Blackglade thats a really nice example and something to work towards. right now i think the primary audience for this is internal as most people just didn't know how money was distributed, and so that we could make better decisions about where we spend money. then i think something like that static viz would be perfect going forward!

**Comment by shapironick** [comment]
![financial-overview 1](https://user-images.githubusercontent.com/7503343/29185791-66507b52-7dd8-11e7-95c2-2bed0efa4d09.jpg)

Here is the latest. I'll present it tomorrow at the weekly meeting.

**Comment by shapironick** [comment]
Stephanie noted on the phone just now that we should note where reoccurring costs lie. It may be complicated as most costs are not disambiguated like in the administrative section.

**Comment by dcwalk** [comment]
agreed! I'm wondering if we want something like source data #2 to cover that, and then make these sort of static visualizations aggregate? (i.e., this says "July 2017" so I'm assuming this is total costs until then) I think we'd need a separate visualization to just capture monthly flows?

**Comment by shapironick** [comment]
yay @sknutson47 just joined GitHub!
